lua/theprimeagen/set.lua

```diff
@@ -14,7 +14,7 @@ vim.opt.wrap = false
 
 vim.opt.swapfile = false
 vim.opt.backup = false
-vim.opt.undodir = os.getenv("HOME") .. "/.vim/undodir"
+vim.opt.undodir = vim.fn.expand('~/.vim/undodir')
```

Comment:
I'm new(ish) to this vim stuff.  Been using an emulator in other IDEs and decided to finally take the jump.   I had a variation on this line, but I'm not sure if there are any downsides to my approach:
`vim.opt.undodir = vim.fn.stdpath("data") .. "/undodir"`

I ended up using `stdpath("data")` assuming it would be available on all platforms.